prep-javascript-objects/main.js

```diff
@@ -0,0 +1,47 @@
+var person = {
+  firstName: 'Rocky',
+  lastName: 'Truong',
+  hobby: 'tennis'
+};
+
+console.log(person);
+
+person.job = 'unemployed';
+person.previousJob = 'intern';
+
+var fullName = (person.firstName) + ' ' + (person.lastName);
+
+console.log("The person's name is:", fullName + '.');
+
+console.log("The person's current job is:", (person.job) + '.');
+
+console.log("The person's previous job is:", (person.previousJob) + '.');
+
+console.log('The complete person object:', (person));
+
+var myCar = {
+  make: 'Hyundai',
+  model: 'Ioniq',
+  year: '2018'
+};
+
+console.log('Car Information', myCar);
+
+myCar = {
+  owner: fullName,
+  make: myCar.make,
+  model: myCar.model,
+  year: myCar.year
+};
```

Comment:
This part should be `myCar['owner'] = fullName` instead of assigning a whole new object.